codegen/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "weld-codegen"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Because this is adding a new trait bound to the generated output, plus the api change for the out dir, I think 0.7 is what we'll be on